plugins/server/com.ucasoft/simple-memory-cache/2.0/routing.kt

```diff
@@ -0,0 +1,19 @@
+import com.ucasoft.ktor.simpleCache.cacheOutput
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import kotlin.random.Random
+import kotlin.time.Duration.Companion.seconds
+
+public fun Route.cachedEndpoints() {
+    cacheOutput(2.seconds) {
+        get("/short") {
+            call.respond(Random.nextInt())
```

Comment:
Hey, just one more thing - could you convert the argument here to a String (toString or generating a random sequence)?  Returning an integer here yields a 406 Not Acceptable response.

Reply:
Done.